internal/services/servicebus/servicebus_queue_data_source.go

```diff
@@ -37,25 +36,6 @@ func dataSourceServiceBusQueue() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ValidateFunc: namespaces.ValidateNamespaceID,
-				AtLeastOneOf: []string{"namespace_id", "resource_group_name", "namespace_name"},
-			},
-
-			// TODO Remove in 4.0
-			"namespace_name": {
```

Comment:
Unfortunately, these were missed in our 4.0 major release so they still currently work in 4.0. We can't remove them now because if they work in some version of 4.0, it's a breaking change if we remove it.

 We'll need to not only keep them in but also wrap them in 5.0 logic so we can remove it properly for 5.0